src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintrAPIClientTests.kt

```diff
@@ -67,4 +67,20 @@ class HintrApiClientTests {
 
         JSONValidator().validateSuccess(metadataResult.body!!, "PlottingMetadataResponse")
     }
+
+    @Test
+    fun `can download spectrum data`() {
+        val sut = HintrAPIClient(ConfiguredAppProperties(), ObjectMapper())
+        val result = sut.downloadSpectrum("ef5b621dfd9beb8851fcd7de2d6c1e08");
+
+        assertThat(result.statusCodeValue).isEqualTo(400)
+    }
+
+    @Test
+    fun `can download summary data`() {
+        val sut = HintrAPIClient(ConfiguredAppProperties(), ObjectMapper())
+        val result = sut.downloadSummary("ef5b621dfd9beb8851fcd7de2d6c1e08");
+
+        assertThat(result.statusCodeValue).isEqualTo(400)
```

Comment:
What are these 2 tests testing?

Reply:
I guess not a great deal, since unlike the other tests it isn't checking JSON schema, but only that you're getting a 400 back from hintr rather than a 500 or 404. Could change it so that it waits for a run to complete and gets actual successful download data? That's already happening in DownloadTests though.

Reply:
Getting a 400 here is consistent with it being the wrong hintr url, so I don't think these add anything unless there's some way of confirming that the correct url has been hit (maybe the error message has something in it that identifies the endpoint?)